Maquetado/scss/login.scss

```diff
@@ -0,0 +1,71 @@
+@mixin login-item($margin) {
+  @extend %login-text;
+  margin: $margin;
+  width: 100%;
+}
+
+%login-text {
+  border-radius: 5px;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.login-box {
+  align-items: center;
+  background-color: $wild-sand;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  @include border-top-cerulean();
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 25px;
+  position: relative;
+  top: 180px;
+  width: 300px;
+}
+
+.login-box-logo {
+  height: 70px;
+  margin-bottom: 10px;
+}
+
+.login-box-text {
+  @extend %login-text;
+  margin-bottom: 5px;
+}
+
+.login-box-input {
+  @extend %login-text;
+  height: 35px;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  @include login-item(5px 0);
+}
+
+.login-box-btn-log {
+  background-color: $earls-green;
+  color: $white;
+  height: 30px;
+  @include login-item(0 0 15px 0);
+}
+
+.login-box-btn-sign {
+  border: 2px solid  $earls-green;
+  color: $earls-green;
+  height: 30px;
+  @include login-item(15px 0 0 0);
+}
+
+.login-box-divider {
+  border: 1px solid $silver-chalice;
+  width: 100%;
+}
+
+.form-buttons {
+  align-self: stretch;
```

Comment:
Why you use stretch instead of set a height?

Reply:
No specific reason, is there any diference?

Reply:
Yes, `stretch` stretches the children to container `max-height` and `max-width` properties. And `height` set the element height individually, this height is fixed, isn't variable.

Reply:
So, in this case should i put `height: 100%` besides `width: 100%` ?

Reply:
If you want that your buttons adapt his size to container leave the stretch. If not, then replaced by a fixed height and width.